optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -514,7 +515,9 @@ private XYPlot createScalabilityPlot(List<XYSeries> seriesList,
             String yAxisLabel, NumberFormat yAxisNumberFormat) {
         NumberAxis xAxis = new NumberAxis(xAxisLabel);
         if (useLogarithmicProblemScale(seriesList)) {
-            xAxis = new LogarithmicAxis(xAxis.getLabel() + " (logarithmic)");
+            LogarithmicAxis xLogAxis = new LogarithmicAxis(xAxis.getLabel() + " (logarithmic)");
+            xLogAxis.setAllowNegativesFlag(true);
+            xAxis = xLogAxis;
```

Comment:
No need to use a separate xLogAxis variable (especially as it's a local variable, so there is no multi-threading danger whatsoever). Also change the original code to avoid unnecessary instantiation:

NumberAxis xAxis;
if (!...) {
  xAxis = new NumberAxis(xAxisLabel);
} else {
  xAxis = new LogarithmicAxis(...)
  ...
}